assets/js/pages/topic.js

```diff
@@ -22,7 +22,7 @@ window.bootTopic = function () {
   };
 
 
-  if ($("#postExampleText").length >= 1) {
+  if ($("#postExampleText").length >= 1 && !(urlParam("viewer_pane")==1)) {
```

Comment:
It's also to display the `SimpleMDE` widget that allow to write mardown in a nice editor instead of a simple textarea. Can you make sure that the widget also appear in RStudio ?